### DISCUSSION TITLE
Disable all auto closing of quotes , brackets, etc.

### DISCUSSION THREAD

**Discussion by kaykurokawa** [LunarVim/LunarVim#4588] [Q&A] [answered]
Hi sorry for basic question, 

What plug in is responsible for the auto closing of quotes and brackets (i.e, typing [ will automatically create a ] after the cursor )  and how can I disable it ? Coming from basic vim and I just find it to be super annoying.

**Comment by juicy-g** [comment, marked as answer]
The plugin is `nvim-autopairs` and to disable it: `lvim.builtin.autopairs.active = false`.
